.github/ISSUE_TEMPLATE/bug-report.md

```diff
@@ -0,0 +1,47 @@
+---
+name: Bug report
+about: Unexpected behavior? Tell us more
+title: ''
+labels: bug
+assignees: ''
+
+---
+
+# Describe your issue
```

Comment:
It seems like the description and the  `{ 'steps to reproduce', 'expected behavior', 'actual behavior' }` set have a lot in common.
I'm not sure of the best solution, maybe we can just let the issue authors decide what they want to emphasize more, and leave them as they are

Reply:
In https://github.com/inmanta/inmanta/pull/2416/commits/97df07c2c9d53836b23046f865629bacef6439c7 I removed the actual behavior and suggested that this would come at the end of steps to reproduce, is this better?